README.md

```diff
@@ -48,10 +48,19 @@ If you prefer the command line way of downloading and installing things, then fe
   ##### `git clone https://github.com/lugenx/ecohabit.git`
 - Open the cloned folder in VS Code. Open the terminal and make sure its pointing to the root of the cloned project.
 
+
+- Database creation
```

Comment:
when reading from the start of the "how to run" section, all the sentences starts with verbs (Clone the repo..., Open the cloned folder...), it feels like this should be like this: "Create a database for development use"